README.md

```diff
@@ -50,10 +53,13 @@ echo 'eval "$(rbenv init -)"' >> ~/.bash_profile
 source ~/.bash_profile
 
 # ruby (.ruby-version default)
-rbenv install
+rbenv install 2.1.2
```

Comment:
I think that if .ruby-version is present in the directory you run `rbenv install` in (which it is if you're in a checkout for this project) `rbenv install` is technically more correct than `rbenv install 2.1.2`. That said, the ordering on this is a little wonky, so maybe good to specify